kinto/core/cache/memcached.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from functools import wraps
+from math import ceil, floor
+from time import time
+
+import memcache
+from pyramid.settings import aslist
+
+from kinto.core.cache import CacheBase
+from kinto.core.storage import exceptions
+from kinto.core.utils import json
+
+logger = logging.getLogger(__name__)
+
+
+def wrap_memcached_error(func):
+    @wraps(func)
+    def wrapped(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except TypeError:
+            raise
+        except Exception as e:
+            logger.exception(e)
+            raise exceptions.BackendError(original=e)
```

Comment:
What is this about? Is there no Memcache-specific exception parent class we can use here?

Reply:
I didn't find one unfortunately :(